crates/context_aware_config/src/api/config/handlers.rs

```diff
@@ -527,9 +531,8 @@ async fn get(
         config = filter_config_by_dimensions(&config, &query_params_map)?
     }
 
-    let resp = HttpResponse::Ok().json(config);
+    let resp = resp.json(config);
     let audit_resp = add_audit_header(&mut conn, resp)?;
-
     add_last_modified_header(max_created_at, audit_resp)
```

Comment:
Here at the bottom get all the headers in a map and use the type you made to construct the response.

Reply:
yes will do it like that only